### ISSUE TITLE
Sound cuts out randomly after opening client on Linux (ALSA)

### ISSUE THREAD

**Issue by MitchWeaver** [runelite/runelite#4891]
Been dealing with this for months, keep bringing it up in the discord but get brushed off.

Using pure ALSA, sound mixing using `dmix`, the sound will randomly cut out after having the client open for a period of time. The client will need be re-opened to restore it.

-----

inb4 use pulseaudio: no

Also this never happens with OSBuddy, or other java programs as some have suggested.

----

edit: This is the same problem I believe https://github.com/runelite/runelite/issues/3525

Please fix this.

**Comment by deathbeam** [comment]
Can you post output of `java -jar RuneLite.jar --debug`?

**Comment by MitchWeaver** [comment]
@deathbeam http://ix.io/1k78

**Comment by deathbeam** [comment]
Can you upgrade your launcher to new one? that one is quite old. And run the same command again.

**Comment by MitchWeaver** [comment]
newer version: http://ix.io/1k7x

**Comment by deathbeam** [comment]
So it looks like your JVM is outdated. Update it to latest version. Also looks like you don't even have notify-send installed?

**Comment by deathbeam** [comment]
Any update after updating JVM?

**Comment by ase1590** [comment]
I am experiencing a similar issue, as described in #6345  however I **do** have Pulseaudio (and pulseaudio-alsa) installed. Music will never play. 

runelite --debug output: http://ix.io/1qXK

**Comment by ase1590** [comment]
@deathbeam I have all of those packages installed other than alsa-oss. installing alsa-oss made no difference.

**Comment by deathbeam** [comment]
Duplicate of #3525 looks like

**Comment by MitchWeaver** [comment]
> I just tested Runelite on Arch on my laptop, it was suffering the same fate.
> 
> I have found exactly where the break occured. upgrading `alsa-lib` on arch from `1.1.6-1` to `1.1.7-1` causes the music and sound cut out issue to occur. downgrading to 1.1.6-1 seems to be a 'workaround' for now.

OH MY GOD THANK YOU. This resolved my issue!

Since 1.1.7 is the new default for most distros, (certainly was for mine; and its been a while), I think this would be a good thing to add to the wiki next to the pulseaudio-alsa warning @deathbeam

**Comment by lupusbytes** [comment]
Using Gentoo Linux, my RuneLite has been without sound since the 24th of april, when the stable alsa-lib version was bumped from 1.1.6 to 1.1.8. 
I have tried to downgrade, back to 1.1.6, with success, but then soon dependency conflicts started emerge with the stable pulseaudio version etc.
I checked out the 1.1.7 patch some of you have been posting and that is exactly how it already is in 1.1.8. and 1.1.9 (https://git.alsa-project.org/?p=alsa-lib.git;a=blob;f=src/pcm/interval_inline.h;h=d9a30b2a86464376070e54ba10a9b6e1fdd2a44c;hb=108a2f43791fe769fd58726881b12ad908ebd2e0)
Now I'm really confused as to why RuneLite sound is not getting picked up anymore.
